requirements-dev-minimal.txt

```diff
@@ -1,3 +1,4 @@
+appdirs
```

Comment:
just a note: the -minimal files should only contain libraries that we directly import - and it looks like this is most likely a transitive dependency